es-app/src/views/gamelist/GridGameListView.h

```diff
@@ -43,6 +44,10 @@ class GridGameListView : public ISimpleGameListView
 
 	TextComponent mLblRating, mLblReleaseDate, mLblDeveloper, mLblPublisher, mLblGenre, mLblPlayers, mLblLastPlayed, mLblPlayCount;
 
+	ImageComponent mMarquee;
+	VideoComponent* mVideo;
+	bool		mVideoPlaying;
```

Comment:
Can we remove extra tabs from `mVideoPlaying`?

Reply:
Done